src/Nodes/InterfaceLabelNode.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import axios from "axios";
+import * as path from "path";
+import * as vscode from "vscode";
+import { Constants } from "../constants";
+import { TelemetryClient } from "../telemetryClient";
+import { Utility } from "../utility";
+import { DeviceNode } from "./DeviceNode";
+import { InfoNode } from "./InfoNode";
+import { INode } from "./INode";
+import { InterfaceNode } from "./InterfaceNode";
+
+export class InterfaceLabelNode implements INode {
+    constructor(public deviceNode: DeviceNode) {
+    }
+
+    public getTreeItem(): vscode.TreeItem {
+        return {
+            label: "Interfaces",
+            collapsibleState: vscode.TreeItemCollapsibleState.Collapsed,
+            contextValue: "interfaces-label",
+        };
+    }
+
+    public async getChildren(context: vscode.ExtensionContext, iotHubConnectionString: string): Promise<INode[]> {
+        TelemetryClient.sendEvent(Constants.IoTHubAILoadInterfacesTreeStartEvent);
+
+        try {
+            const interfaces = (await axios.get(`https://${Utility.getHostName(iotHubConnectionString)}/digitalTwins/${this.deviceNode.deviceId}/interfaces?api-version=2019-07-01-preview`, {

```

Comment:
better to extract the api-version to const. And also to extract the logic to get the endpoint. Besides, we may also keep an eye on the SDK when it supports this api.

Reply:
Updated.
And Node SDK does not support this for now.